pkg/watermark/publish/options.go

```diff
@@ -34,6 +34,10 @@ type publishOptions struct {
 	// Whether it is sink publisher or not
 	// isSource and isSink should not be both true
 	isSink bool
+	// isReduce to indicate it is reduce publisher
+	isReduce bool
+	// toVertexPartition to indicate the partition of toVertex buffer
```

Comment:
Also mention it's only useful when `isToVertexReduce` is true.

Reply:
this should be true for everything. that means we will be adding partition info for map too.